.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,23 @@
+name: gitleaks
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
```

Comment:
Any chance to have these branches set to anything but "develop" and "main" ? this way we see issues before were are ready to merge PRs into develop

Reply:
@atulsriv that is what the precommit hook is for. It will prevent you from committing in your local if there is any issue.

Reply:
right, but does this only apply to the 2 branches listed here? 

i.e if I am on `testbranch`, will i be under the same restrictions?

Reply:
@atulsriv For the Github workflow, we want to scan it daily and eventually send a report to security@bloxroute.com. Being able to do that from GHA won't be feasible because you have to add the branch name. However, the precommit hook will work on your local to any branch. It works on commits, not branches.

Reply:
Okay yeah after looking at the readme makes sense